index.bs

```diff
@@ -126,30 +147,44 @@ spec: mediaqueries-4; urlPrefix: https://drafts.csswg.org/mediaqueries-4/#
     </section>
 
     <section>
-      <h4 id='mediaconfigurationtype'>MediaConfigurationType</h4>
+      <h4 id='mediadecodingtype'>MediaDecodingType</h4>
 
       <pre class='idl'>
-        enum MediaConfigurationType {
+        enum MediaDecodingType {
           "file",
           "media-source",
         };
       </pre>
 
       <p>
-        A {{MediaConfiguration}} has two types:
+        A {{MediaDecodingConfiguration}} has two types:
         <ul>
-          <li><dfn for='MediaConfigurationType' enum-value>file</dfn> is used to
+          <li><dfn for='MediaDecodingType' enum-value>file</dfn> is used to
           represent a configuration that is meant to be used for a plain file
           playback.</li>
-          <li><dfn for='MediaConfigurationType' enum-value>media-source</dfn> is
-          used to represent a configuration that is meant to be used for
-          playback of a {{MediaSource/MediaSource}} as defined in the
-          [[media-source]] specification.</li>
+          <li><dfn for='MediaDecodingType' enum-value>media-source</dfn> is used
+          to represent a configuration that is meant to be used for playback of
+          a {{MediaSource/MediaSource}} as defined in the [[media-source]]
+          specification.</li>
         </ul>
       </p>
     </section>
 
     <section>
+      <h4 id='mediaencodingtype'>MediaEncodingType</h4>
+
+      <pre class='idl'>
+        enum MediaEncodingType {
+          "todo",
```

Comment:
You could add already here a `"local"` entry, but I'm fine to leave it to a later PR too.